src/super_gradients/common/sg_loggers/dagshub_sg_logger.py

```diff
@@ -165,26 +165,51 @@ def _dvc_add(self, local_path="", remote_path=""):
     def _dvc_commit(self, commit=""):
         self.dvc_folder.commit(commit, versioning="dvc", force=True)
 
+    @multi_process_safe
+    def _get_nested_dict_values(self, d, parent_key="", sep="/"):
+        items = []
+        for k, v in d.items():
+            new_key = parent_key + sep + k if parent_key else k
+            if isinstance(v, dict):
```

Comment:
please use `isinstance(v, Maping)` instead.
some of the dictionaries might be DictConfig

Reply:
Fixed this comment @ofrimasad sorry for the delayed response.